i18n/locales/en.json

```diff
@@ -59,7 +59,7 @@
         "healthcareProfessionalName": "Healthcare Professional Name",
         "invalidOption": "Invalid option",
         "lastName": "Last Name",
-        "lastNameValidation": "Enter a last name",
+        "nameValidation": "Enter a first and last name",
```

Comment:
if you split the names, I would make a separate string here for `first` and `last`

Reply:
I separated them to be different for each one. And added the values to all the languages `.json` files. The Portuguese and Japanese have the english and will need to be updated later. I think maybe we should have a way to have a bot to create a ticket in the CI/CD that anytime a language file is merged a ticket is made to check to see if the languages files have any untranslated updates. If the person who wrote the ticket translated files cuz they have the fluency we can close the issue. Otherwise it can be medium priority? @Anissa3005 @ermish @theyokohamalife ?